build.sbt

```diff
@@ -36,16 +38,26 @@ libraryDependencies ++= Seq(
 
 scalacOptions ++= Seq("-feature")
 
+javaOptions in Universal ++= Seq(
+  s"-Dpidfile.path=/dev/null"
```

Comment:
Related to how Play handles pid file: sbt-native-packager maintainer recommends to let play not handle the pid file.